src/main/kotlin/com/cognifide/gradle/aem/common/instance/local/QuickstartResolver.kt

```diff
@@ -29,6 +36,9 @@ class QuickstartResolver(private val aem: AemExtension) {
      */
     val licenseUrl = aem.obj.string {
         aem.prop.string("localInstance.quickstart.licenseUrl")?.let { set(it) }
+        aem.prop.string("localInstance.quickstart.dirUrl")?.let {
+            set(File(it).walk().filter { it.name.equals("license.properties") }.first().absolutePath)
```

Comment:
if walk is recursive this is potentially dangerous, maybe listFiles() will be safer

Reply:
I mean when some subdir will contain huge no of files